pandas/tests/test_dtype_utc.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Sat Dec 17 09:34:47 2016
+
+@author: rodolfoxps
+"""
+import nose
+import pandas as pd
+import datetime
+import pytz
+from pandas.util.testing import assert_series_equal
+
+
+def test_dtype_utc():
+
+    data = pd.Series([pd.NaT, pd.NaT,
+                      datetime.datetime(2016, 12, 12, 22, 24, 6, 100001,
+                                        tzinfo=pytz.utc)])
+
+    filled = data.fillna(method='bfill')
+
+    expected = pd.Series([datetime.datetime(2016, 12, 12, 22, 24, 6, 100001,
```

Comment:
this test needs to go in test_missing (search and out it with other similar tests)